Fix prettier spelling. (#2)
Andymendez100/Typescript-Template: README.md

> # Typescript-Template

Reviewer: ```suggestion
# TypeScript-Template
```

> Starting points for Typescript with Airbnb Eslint, Husky with lint-staged. and Prettier

Reviewer: ```suggestion
Starter template using TypeScript, ESLint with Airbnb configuration, Husky configured with Lint-staged and Prettier.
```

> Frontend: CRA with typescript, Airbnb Eslint

Reviewer: ```suggestion
Frontend: CRA with TypeScript, ESLint with Airbnb configuration.
```
